internal/binary.bzl

```diff
@@ -31,6 +31,7 @@ def postcss_binary(
         additional_outputs = [],
         output_name = "",
         map_annotation = False,
+        data = [],
```

Comment:
Is data not passed along by kwargs?

Reply:
We need to make sure it only goes to `postcss_run`, not `postcss_gen_runner`.